rsmtool/notebooks/comparison/intermediate_file_paths.ipynb

```diff
@@ -0,0 +1,68 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Links to Intermediate Files"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Click on the hyperlinks below to see the intermediate experiment files output as part of this comparison."
+   ]
```

Comment:
My biggest concern here is that when the users share that report they probably won't be sharing the original directories or if they do the paths would change when they zip everything up. I'd probably just keep the skeleton for future comparison outputs and not share the paths to the original files.

Reply:
Actually, the links in the current reports generated by test (I am looking at `lr-self-compare-different-formats`) do not work so it will be a problem.

Reply:
Ah, I see the problem. The relative paths aren't working properly for the `rsmcompare` and `rsmsummary`.I can fix that, but should we just get rid of the links instead? We can change the title to "List of intermediate files" or something?

Reply:
I would vote for commenting out the notebook for `RSMCompare` for the time being (or just keeping it empty) since there are no outputs generated (we'll want to add it back when we address the outputs issue so please don't delete all the great code you already wrote!). 

 For `RSMSummarize` we can just give the links to `rsmsummarize` outputs in the `output` folder - hopefully this is a straightforward change.

What do you think?

Reply:
Yup! Makes sense!